frontend/src/reducers/panelReducer.ts

```diff
@@ -28,29 +30,30 @@ export const panelSlice = createSlice({
         // Sets all the panels to new ones
         setPanels(state, action: PayloadAction<ContentPanelData[]>) {
             state.value = action.payload;
```

Comment:
needsSaving seems to get assigned false and true a bit arbitrarily. Why does setPanels not need saving? What if importing panels is later implemented using setPanels? This also seems to be a fix to a more underlying issue, which is the ContentPanel autosave/save functionality.